src/renderer/routes.ts

```diff
@@ -69,6 +78,7 @@ export const routes: RouteInfo[] = [
     planet,
     moon,
     comet,
+    devship,
```

Comment:
I think dev ships should come last, it's less likely than the other options imo.